src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -433,7 +437,20 @@ Whether Harvesting (OAI) service is enabled
          * Installation Brand Name is always included (default/false) or is not included
          * when the Distributor field (citation metadatablock) is set (true)
          */
-        ExportInstallationAsDistributorOnlyWhenNotSet
+        ExportInstallationAsDistributorOnlyWhenNotSet,
+
+        /**
+         * BasicGlobusToken for Globus Application
+         */
+        BasicGlobusToken,
```

Comment:
FWIW: For DataCite the credentials are jvm-options, whereas Amazon's S3 client picks them up from the standard files in the unix user's .aws subdir (and we have a jvm-option to specify which 'profile' to use in those files.)
In general, the settings are for things where you might want to change on the fly and are potentially remotely accessible if you don't block non-local access.